tests/console/validate_raid_encrypt_home.pm

```diff
@@ -0,0 +1,92 @@
+# SUSE's openQA tests
```

Comment:
we talked in person about this :) please don't use the old approach to validate raid partitioning, it is hardly maintainable, we just need to be sure that the encrypted home is part of the multi-disk (md) running some command.

Reply:
Updated.